src/lib.ts

```diff
@@ -517,17 +517,12 @@ function coldMedicineCabinet(): void {
 
 function horseradish(): void {
   if (myFullness() < fullnessLimit()) {
-    if (mallPrice($item`fudge spork`) < 3 * get("valueOfAdventure") && !get("_fudgeSporkUsed"))
-      eat(1, $item`fudge spork`);
-    MayoClinic.setMayoMinder(MayoClinic.Mayo.zapine, 1);
-    acquire(1, $item`Special Seasoning`, get("valueOfAdventure"));
-    acquire(1, $item`jumping horseradish`, 5.5 * get("valueOfAdventure"));
-    if (!eat(1, $item`jumping horseradish`)) throw "Failed to eat safely";
+    fillPantsgiving();
   }
 }
 
-export function postCombatActions(): void {
-  horseradish();
+export function postCombatActions(skipDiet = false): void {
+  if (!skipDiet) horseradish();
```

Comment:
I'd rather handle this in the horseradish function. also I'm not sure why we have a separate `fillPantsgiving()` function from the `horseradish()` function--why not excise horseradish entirely and just use `fillPantsgiving`?

Reply:
this is a good suggestion - it should just call the diet's "pantsgiving" directly instead.